docs/usage/overview.md

```diff
@@ -99,6 +99,8 @@ Some of the configuration options which can be used to generate the CMake projec
 
 - `CLFML_FACE_DETECTOR_ENABLE_CORAL_TPU`; Enables Coral TPU support (Default=OFF)
 
+- `CLFML_ROS2_PACKAGE_BUILD`; Build a ROS2 (Default=OFF, *Note that colcon is used as the build system.) 
```

Comment:
Here it states that it is default OFF, but in the CMakeLists it was defined as default ON